app/src/main/res/menu/side_drawer_menu.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8"?>
+<menu xmlns:android="http://schemas.android.com/apk/res/android">
+    <item
+        android:id="@+id/mytasks"
+        android:title="@string/drawer_my_tasks" />
+
+
+    <item android:title="HEIG-VD">
+        <menu>
+            <item
+                android:id="@+id/test"
+                android:title="   GEN" />
+            <item
+                android:id="@+id/test2"
+                android:title="   SIO" />
+            <item
+                android:id="@+id/test3"
+                android:title="   POO2" />
+            <item
+                android:id="@+id/test4"
+                android:title="   RES" />
+            <item
+                android:id="@+id/test5"
+                android:title="   SER" />
+        </menu>
+    </item>
```

Comment:
As the data seems to be added correctly these test elements should be gone.

Reply:
> OK my drawer is gone then I guess. I find the fact that you can't minimise the folder problematics aswith few folders this is ok but with a lot of them it's not.

Yes I think we should use the tools android gives us, it makes things easier than to try and make our custom views... We can talk more if you want to go back to the way it was. Will look into the collapsing folder feature.

Reply:
I think you're right. But you should have at least consulted me on the matter. Generally if we change something that someone has done in a drastic way it's better to consult that person beforehand.

Reply:
Alright noted, I thought the PR was enough.

Reply:
It's really pretty tho :3